pygame_gui/elements/ui_drop_down_menu.py

```diff
@@ -627,7 +636,16 @@ class UIDropDownMenu(UIContainer):
     The drop down is implemented through two states, one representing the 'closed' menu state
     and one for when it has been 'expanded'.
 
-    :param options_list: The list of of options to choose from. They must be strings.
+    :param options_list: The list of options to choose from. They must be strings.
```

Comment:
shame I didn't name this parameter just 'options' eh? But I guess it is a bit too late now.